From a4c482fb6a99b74be86db8efdb02d5141c65e2fc Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Tue, 17 Jun 2025 18:38:24 +0300
Subject: [PATCH] [JSW-670] jgs: address space update

Updated palad boot loader address to run from
psoc rom memory as per JGS.

Change-Id: I0b9a16c7dee7c52fb3048ae366fb51c8e79bd4d2
---

diff --git a/arch/jgs/arc/palad_bloader.lds.S b/arch/jgs/arc/palad_bloader.lds.S
index ec3bb56..255bc90 100644
--- a/arch/jgs/arc/palad_bloader.lds.S
+++ b/arch/jgs/arc/palad_bloader.lds.S
@@ -9,7 +9,7 @@
 MEMORY
 {
 	ICCM	: ORIGIN = 0x00000000, LENGTH = 256K
-	SBROM	: ORIGIN = 0x78980000, LENGTH = 128K
+	SBROM	: ORIGIN = 0x78620000, LENGTH = 128K
 	DCCM	: ORIGIN = 0x80000000, LENGTH = 128K
 }
 
